fastlane/Fastfile

```diff
@@ -32,6 +32,32 @@ platform :android do
         )
     end
 
+
+    desc "Publish the app to Huawei AppGallery"
+    desc "#### Example:"
+    desc "```\nfastlane android publish_app_gallery client_id:xxx client_secret:xxx app_id:xxx apk_path:xxx\n```"
+    desc "#### Options"
+    desc " * **`client_id`**: The client ID for Huawei AppGallery Connect"
+    desc " * **`client_secret`**: The client secret for Huawei AppGallery Connect"
+    desc " * **`app_id`**: The app ID for the application"
+    desc " * **`apk_path`**: The path to the APK/AAB file"
+    desc " * **`organization`**: ooni, dw (default: ooni)"
+    desc ""
+    lane :publish_app_gallery do |options|
+        organization = options[:organization] || "ooni"
+        raise "Invalid organization #{organization}" unless %w(ooni).include?(organization)
+
```

Comment:
It's missing the `bundle` fastlane method from the other PR right?

Reply:
Yeah. I saw that in the other change set and didnt want to duplicate it here